### PR TITLE
#132. Added breadcrumbs.

### PR DESCRIPTION
**Pull request by serhii-morozov**
commerce_draft-132: Added breadcrumbs - easy_breadcrumbs module. Added product categories to breadcrumbs.

### CODE REVIEW

File: config/default/block.block.breadcrumbs.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+langcode: en
+status: true
+dependencies:
+  module:
+    - system
+  theme:
+    - olivero
+id: breadcrumbs
+theme: olivero
+region: header
```
Comment (Macsym2018) [inline config/default/block.block.breadcrumbs.yml:11]: It is better to display breadcrumbs in the breadcrumbs region.
And maybe change visibility only for Commerce Product.

**Review by Macsym2018 — CHANGES_REQUESTED** [review]
It is better to display breadcrumbs in the breadcrumbs region.
Maybe you need to make the block visible only for the productpage.

**Comment by Macsym2018** [comment]
> 
@serhii-morozov 
There is the "Breadcrumb" region in Block Layout by default.

File: docroot/modules/custom/draft_wishlist/draft_wishlist.info.yml
Comment (ozin7) [inline docroot/modules/custom/draft_wishlist/draft_wishlist.info.yml:1]: You remove this module but this module is not disabled in core.extention.
If you want to remove module you have to disable it first, then deploy this to prod and then with next deploy you can remove the module code

**Review by ozin7 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**